packages/contracts-bedrock/src/libraries/EOA.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/// @title EOA
+/// @notice A library for detecting if an address is an EOA.
```

Comment:
Mixing named return with explicit return is weird

Reply:
Updated to use named return only, and brought back the else statement